src/guides/v2.3/ext-best-practices/tutorials/dynamic-row-system-config.md

```diff
@@ -171,7 +171,67 @@ class TaxColumn extends Select
 
 This block sets values for the drop-down option.
 
-## Step 4: Clean cache
+## Step 4: Set default values - **OPTIONAL**
+It is possible to set defaults for a dynamic row configuration, this is done by adding additional XML to the defaults block in the `config.xml` file for the module.
+
+Add a block to the `<default>` section in the file as it would be done normally but do not add any values. For the example this tutorial has been using, this would look like :
+
+```xml
+<system>
+    <general>
+        <ranges></ranges>
+    </general>
+</system>
+```
+
+For each complete line of configuration, create an XML block with a repeating node that has a different value to all the others and contains XML tags for each sub-option and value.
+So we could use `<item1>, <item2>` etc.
+
+The sub-options are the columns defined in the `_prepareToRender()` method as described in [Step 2](#step-2-create-the-block-class-to-describe-custom-field-columns).
+
+So for a single row for above we may have :
+
+```xml
+<item1>
+    <from_qty>5</from_qty>
+    <to_qty>6</to_qty>
+    <price>10.00</price>
+    <tax>1</tax>
+</item1>
+```
+
+Putting this all together the default block for the configuration option 'ranges', with 3 values could look like :
+
```

Comment:
For this code block, you can use the convention (used earlier in this topic) of indicating in which file to add the code. I think using `>`